addons/website/views/snippets/s_product_catalog.xml

```diff
@@ -2,15 +2,15 @@
 <odoo>
 
 <template id="s_product_catalog" name="Product Catalog">
-    <section class="s_product_catalog oe_img_bg oe_custom_bg pt48 pb48"
+    <section class="s_product_catalog oe_img_bg oe_custom_bg pt48 pb32"
              style="background-image: url('/web/image/website.s_product_catalog_default_image');">
         <div class="container">
             <h2>Menu</h2>
-            <h5>Add a menu description</h5>
+            <p class="lead">Add a menu description.</p>
             <div class="row">
-                <div class="col-lg-6 pt24">
+                <div class="col-lg-6 pt16 pb16">
                     <h3>Starter</h3>
-                    <div class="row">
+                    <div class="row s_col_no_resize">
```

Comment:
It's hard to edit the content with all the resizing arrows. Especially, when there is no description.
[https://drive.google.com/open?id=1gStp29z5G3A5QxvLk1Lft5iAc6eGjDIo](https://drive.google.com/open?id=1gStp29z5G3A5QxvLk1Lft5iAc6eGjDIo)
I don't see the point to be able to resize all the lines one by one.
Sorry, I'm missing something here. Why is it useful?

Reply:
I agree with @Cocographique on this one.
This way user won't do anything stupid :-)

Reply:
I don't think this:
![Screenshot from 2020-04-20 12-10-55](https://user-images.githubusercontent.com/10338094/79740512-14c64c00-8300-11ea-936a-0cec842a48af.png)
is stupid but as you wish :wink: 

@Cocographique Can you also review the structure/scss so that triple click on an item + try to bold/italic whatever works? Thanks ;)

Reply:
Also, as we don't allow to resize, I guess you can remove the row/col grid entirely instead of marking them as non resizable :+1:

Reply:
No, indeed, your example isn't stupid at all. 
But on this point, I think limiting the customization for the user is a good thing.

Reply:
Triple-click + bold/italic should work now.

About s_col_no_resize, I'm not against the ability to resize the columns. My main concern is about the arrows overlapping of the content. It could be solved by reducing the size of the arrows. But this is another story...
<img width="600" alt="Screenshot 2020-04-20 at 15 22 21" src="https://user-images.githubusercontent.com/7629949/79757236-1b62bc80-831c-11ea-9fe2-cd0aedec1586.png">

Reply:
SBU is probably right, let's go with easing that for the user.

Reply:
If I remove the row/col grid, it's almost impossible to put your cursor at the beginning of the line. Is that okay for you? (Personally, I don't like it).
<img width="200" alt="Screenshot 2020-04-21 at 10 53 10" src="https://user-images.githubusercontent.com/7629949/79846394-b01df680-83be-11ea-9959-3dacb6e579d8.png">

Reply:
We should not have to solve UI problem by altering the snippet DOM so this is not a problem. But I of course agree that this should be fixed another way. Do you have a magic idea @sbuhl ?